src/toolbox/shamir.rs

```diff
@@ -240,27 +250,28 @@ impl SecretShare {
     }
 }
 
+#[allow(unused_imports)]
 mod tests {
-    #[allow(unused_imports)]
     use super::*;
+    use crate::toolbox::secrets::SecretSharing;
 
     #[test]
     fn shamir_pow_easy() {
```

Comment:
I guess the pow-testing should move to util.rs

Reply:
Yep, missed that.  Thanks!